diff -up sbc-1.5/sbc/sbc_primitives.c.omv~ sbc-1.5/sbc/sbc_primitives.c
--- sbc-1.5/sbc/sbc_primitives.c.omv~	2021-01-07 02:00:55.124669682 +0100
+++ sbc-1.5/sbc/sbc_primitives.c	2021-01-06 23:51:30.345401679 +0100
@@ -593,6 +593,7 @@ static int sbc_calc_scalefactors_j(
 
 static void sbc_init_primitives_x86(struct sbc_encoder_state *state)
 {
+#if defined(__x86_64__) || defined(__i386__)
 	__builtin_cpu_init();
 
 #ifdef SBC_BUILD_WITH_MMX_SUPPORT
@@ -604,6 +605,7 @@ static void sbc_init_primitives_x86(stru
 	if (__builtin_cpu_supports("sse4.2"))
 		sbc_init_primitives_sse(state);
 #endif
+#endif
 }
 
 /*
